Domain/Aggreagtes/RoleAggregate/Role.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Domain.Aggreagtes.EnrollmentAggregate;
+using Domain.Aggreagtes.UserAggregate;
+using Domain.Common.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Aggreagtes.RoleAggregate
+{
+    public class Role : AuditableEntity<Guid>, IAggregateRoot
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public ICollection<User> Enrollments { get; private set; } = new HashSet<User>();
```

Comment:
What is the purpose of Collection named Enrollments in Roles.?
A List of users should be Users